dex/src/test/scala/com/wavesplatform/dex/actors/MatcherActorSpecification.scala

```diff
@@ -41,10 +41,12 @@ class MatcherActorSpecification
     with Eventually
     with SystemTime {
 
+  private val twoHundredMillisTimeout = 200.millis
```

Comment:
let's make it just "timeout". I already understand that it's "200.millis" looking at its impl)

Reply:
btw is 200 millis sufficient here?

Reply:
It's not implicit timeout, it's timeout for expectNoMsg. It was like this:
```scala
          probe23.expectNoMessage(200.millis)
          probe45.expectNoMessage(200.millis)
```
And now it's like that
```scala
          probe23.expectNoMessage(noMsgTimeout)
          probe45.expectNoMessage(noMsgTimeout)
```
(I renamed it to `noMsgTimeout` in fix commit)